awx/ui/client/lib/models/Inventory.js

```diff
@@ -0,0 +1,44 @@
+let BaseModel;
+let JobTemplateModel;
+let ModelsStrings;
+
+function setDependentResources (id) {
+    this.dependentResources = [
+        {
+            model: new JobTemplateModel(),
+            params: {
+                inventory: id
+            }
+        }
+    ];
+}
+
+function InventoryModel (method, resource, graft) {
```

Comment:
I think this third constructor argument should become `config`, which means BaseModel's `create` would have to get updated as well. It'd make sense now that we're packing more functionality into these models.